models/contacts.js

```diff
@@ -1,19 +1,86 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const path = require("path");
+const { v4: uuidv4 } = require("uuid");
 
-const listContacts = async () => {}
+const contactsPath = path.resolve("./models/contacts.json");
 
-const getContactById = async (contactId) => {}
+async function listContacts() {
+  try {
+    const data = await fs.readFile(contactsPath);
+    const contacts = JSON.parse(data);
+    console.table(contacts);
+    return contacts;
+  } catch (error) {
+    console.error(error);
+  }
+}
 
-const removeContact = async (contactId) => {}
+async function getContactById(contactId) {
+  try {
+    const data = await fs.readFile(contactsPath);
```

Comment:
instead of duplicate already existing logic, try to use `await listContacts()` for fetching all contacts in other methods? What do you think about it?